packages/keybr-textinput/lib/settings.ts

```diff
@@ -59,10 +60,15 @@ export enum WhitespaceStyle {
   Bar = 2,
   Bullet = 3,
 }
+export enum EnableSoundStyle {
+  All = 1,
+  Error = 2,
+}
 
 export const textDisplaySettings: TextDisplaySettings = {
   caretShapeStyle: CaretShapeStyle.Underline,
   caretMovementStyle: CaretMovementStyle.Smooth,
+  enableSoundStyle: EnableSoundStyle.All,
   whitespaceStyle: WhitespaceStyle.Bullet,
   sounds: false,
```

Comment:
Let's remove these two properties and merge them into one.
So,  remove the `enableSourndsStyle` and replace `sounds` with

```
  playSounds: PlaySounds.None,
```